### ISSUE TITLE
Google Voice voicemail playback is blocked on Chrome (Chromebase)

### ISSUE THREAD

**Issue by ezahduu** [Eloston/disable-html5-autoplay#139]
Both "New" and "Legacy" Google Voice share this. The playback buttons change and work, but it appears there is no download of the voicemail and no sound. Workaround: Changing the extension to "Disable nothing" and reloading page allows play. 
Chrome: Version 56.0.2924.110 (64-bit)
Extension: 0.6.2

**Comment by kuzmeech** [comment]
Yes, this broke my Google Voice incoming sound too :) Realized it after I disabled ton of other things system-wide and rebooted. 
Adding mail.google.com to exclusion list didn't help me, I'll try more when I have time.

Maybe worth having Google Voice in Chrome enabled by default as it was too confusing.
